bin/detect.py

```diff
@@ -18,6 +18,7 @@
 print("Is this a Giant Board?", detector.board.GIANT_BOARD)
 print("Is this a Coral Edge TPU?", detector.board.CORAL_EDGE_TPU_DEV)
 print("Is this a SiFive Unleashed? ", detector.board.SIFIVE_UNLEASHED)
+print("Is this a PYNQ Board?", detector.board.PYNQ_Z1 | detector.board.PYNQ_Z2)
```

Comment:
Why not calling the `any_pynq_board` method? isn't it easier?

Reply:
We do that in the next hunk further down. I'm not entirely sure what the purpose of doing it both ways is - I've simply followed the convention used in the rest of the file, but I assumed it was just to make sure that flags were being set appropriately and that a caller can use either method for checking the board group or specific ID.